libc/utils/MPFRWrapper/MPFRUtils.h

```diff
@@ -271,7 +290,8 @@ class MPFRMatcher : public testing::Matcher<OutputType> {
 // types.
 template <Operation op, typename InputType, typename OutputType>
 constexpr bool is_valid_operation() {
-  return (Operation::BeginUnaryOperationsSingleOutput < op &&
+  return (op == Operation::Sqrt) ||
+         (Operation::BeginUnaryOperationsSingleOutput < op &&
```

Comment:
All of the changes for the exhaustive test need cleanup before they can be merged, and especially this one.